From 2fb60bcdc8913f3189752fb910942b5f090df348 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Tue, 2 Dec 2014 13:32:27 -0500
Subject: [redhat] scripts: cross, update epel location and add additional packages

Message-id: <1417527147-391-1-git-send-email-prarit@redhat.com>
Patchwork-id: 101029
O-Subject: [RHEL7.1 PATCH INTERNAL] cross, update epel location and add additional packages
Bugzilla:
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

EPEL has reorganized its repo structure yet again, so we have to make that
change here.  In addition to that we require two more packages to
successfully build, ncurses-devel and numactl-devel.  diffstat is required
for pretty cross compile output.

Sucessfully tested by me.

Cc: Don Zickus <dzickus@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
Cc: Neil Horman <nhorman@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/Makefile.cross b/redhat/Makefile.cross
index d99a6b7..ee999ea 100644
--- a/redhat/Makefile.cross
+++ b/redhat/Makefile.cross
@@ -1,6 +1,6 @@
 CROSS_RPMFLAGS = $(RPMBUILD) --define "_sourcedir $(SOURCES)" --define "_builddir $(RPM)/BUILD" --define "_srcrpmdir $(RPM)/SRPMS" --define "_rpmdir $(RPM)/RPMS" --define "_specdir $(RPM)/SPECS" --define "dist $(DIST)"
 
-CROSS_PACKAGE_LIST = binutils-powerpc64-linux-gnu binutils-s390x-linux-gnu cross-binutils-common cross-gcc-common gcc-powerpc64-linux-gnu gcc-s390x-linux-gnu glibc-static
+CROSS_PACKAGE_LIST = binutils-powerpc64-linux-gnu binutils-s390x-linux-gnu cross-binutils-common cross-gcc-common gcc-powerpc64-linux-gnu gcc-s390x-linux-gnu glibc-static ncurses-devel numactl-devel diffstat
 
 rh-cross-download:
  @if [ "$(ARCHCONFIG)" != "X86_64" ]; then \
diff --git a/redhat/scripts/download_cross.sh b/redhat/scripts/download_cross.sh
index 81a7a6e..627a205 100755
--- a/redhat/scripts/download_cross.sh
+++ b/redhat/scripts/download_cross.sh
@@ -8,7 +8,7 @@ rpm -q $@ && exit 0
 # install epel-release if necessary
 rpm -q epel-release >& /dev/null
 if [ $? -ne 0 ]; then
- wget -nd -r -l1 --no-parent -A "epel-release*.rpm" http://download.bos.redhat.com/pub/epel/beta/7/x86_64
+ wget -nd -r -l1 --no-parent -A "epel-release*.rpm" http://download.bos.redhat.com/pub/epel/7/x86_64/e/
  rpm -ivh epel-release*.rpm
  # clean up
  rm -f epel-release*.rpm
-- 
1.7.1